src/components/beat.js

```diff
@@ -36,8 +36,8 @@ AFRAME.registerComponent('beat-system', {
   // Will be overridden.
   verticalPositions: {
     bottom: 0.95,
-    middle: 1.35,
-    top: 1.75
+    middle: 1.45,
```

Comment:
These values don't affect, middle/bottom blocks overlap.

Reply:
> 
> 
> I think left and right side blocks feel a tiny bit of a reach for my arm lengths, especially in punch mode (although the distances there need to be adjusted anyways).

I think blocks can be smaller and closer in punch mode. Each mode should define its own parameters.

Reply:
In classic mode, i think it feels a bit of a contortion for the upper right / left notes. also need to adjust for blocks to not overlap

Reply:
Using ratios for blocks height should help bring those closer.

Reply:
we can try ratio, but need to adjust to handle sitting mode too